### DISCUSSION TITLE
Laravel Vue i18n

### DISCUSSION THREAD

**Discussion by xiCO2k** [HassanZahirnia/laravel-package-ocean#136] [Package Suggestions]
laravel-vue-i18n is a Vue3 plugin that allows to connect your Laravel Framework translation files with Vue. It uses the same logic used on [Laravel Localization](https://laravel.com/docs/8.x/localization).

https://github.com/xico2k/laravel-vue-i18n

**Comment by HassanZahirnia** [comment]
Hey 👋 As I mentioned on X, the package already exists so I'm gonna close this request! 😊

**Reply by xiCO2k** [reply]
💪
